doc/en/user/source/configuration/crshandling/coordtransforms.rst

```diff
@@ -209,22 +209,39 @@ Custom NTv2 file::
 
 Geocentric transformation, preceded by an ellipsoid to geocentric conversion, and back geocentric to ellipsoid. The results is a concatenation of three math transforms::
 
-  4230,4258=CONCAT_MT[PARAM_MT["Ellipsoid_To_Geocentric", \
-    PARAMETER["dim", 2], \
-    PARAMETER["semi_major", 6378388.0], \
-    PARAMETER["semi_minor", 6356911.9461279465]], \
-  PARAM_MT["Position Vector transformation (geog2D domain)", \
-    PARAMETER["dx", -116.641], \
-    PARAMETER["dy", -56.931], \
-    PARAMETER["dz", -110.559], \
-    PARAMETER["ex", 0.8925078166311858], \
-    PARAMETER["ey", 0.9207660950870382], \
-    PARAMETER["ez", -0.9166407989620964], \
-    PARAMETER["ppm", -3.5200000000346066]], \
-  PARAM_MT["Geocentric_To_Ellipsoid", \
-    PARAMETER["dim", 2], \
-    PARAMETER["semi_major", 6378137.0], \
-    PARAMETER["semi_minor", 6356752.314140356]]]
+  4230,4258=CONCAT_MT[ \
+    PARAM_MT["Ellipsoid_To_Geocentric", \
+      PARAMETER["dim", 2], \
+      PARAMETER["semi_major", 6378388.0], \
+      PARAMETER["semi_minor", 6356911.9461279465]], \
+    PARAM_MT["Position Vector transformation (geog2D domain)", \
+      PARAMETER["dx", -116.641], \
+      PARAMETER["dy", -56.931], \
+      PARAMETER["dz", -110.559], \
+      PARAMETER["ex", 0.8925078166311858], \
+      PARAMETER["ey", 0.9207660950870382], \
+      PARAMETER["ez", -0.9166407989620964], \
+      PARAMETER["ppm", -3.5200000000346066]], \
+    PARAM_MT["Geocentric_To_Ellipsoid", \
+      PARAMETER["dim", 2], \
+      PARAMETER["semi_major", 6378137.0], \
+      PARAMETER["semi_minor", 6356752.314140356]]]
+
+You can make use of existing grid shift files such as this explicit transformation from NAD27 to WGS84 (made up of a NADCON transform from NAD27 to NAD83 followed by a Molodenski transform from NAD83 to WGS84)::
```

Comment:
I believe the Molodenski xform moves from one ellipsoid to another.  Might better to say "... followed by a Moldenski transform to convert from the GRS80 Ellipsoid (used by NAD83) to the WGS84 Ellipsoid..."